plugins/spiral-html/scripts/audit.sh

```
@@ -0,0 +1,270 @@
+# Prints a remediation checklist. Does not modify anything.
+
+set -uo pipefail
+
+TARGET_DIR="${1:-$(pwd)}"
+
+if [[ ! -d "${TARGET_DIR}" ]]; then
+  echo "Error: ${TARGET_DIR} is not a directory." >&2
+  exit 1
+fi
+
+cd "${TARGET_DIR}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)